src/redux/groups/groupsSliceMiddleware.ts

```diff
@@ -37,6 +36,8 @@ const computeGroups = (
     }
     groups[guiPath] = groups[guiPath] || emptyGroup();
     groups[guiPath].propertyOwners.push(uri);
+
+    // Finally, sort the property owners based on the GuiOrderingNumber
```

Comment:
Woopsie :) I guess that I at some point thought that it would be more useful to sort the nodes directly when generating the data for the groups.

It might still be, but if so, when generating the tree data. That's something to discuss I guess. Currently the sorted and filtered tree data is memoized